package.json

```diff
@@ -15,14 +15,15 @@
   },
   "scripts": {
     "clean": "rm -rf src/*/node_modules/ build/ node_modules/ src/server_manager/install_scripts/do_install_script.ts",
-    "do": "bash ./scripts/do_action.sh"
+    "do": "bash ./scripts/do_action.sh",
+    "lint": "tslint 'src/**/*.ts' -e '**/node_modules/**'"
```

Comment:
Restore `tslint --fix ...`?

Reply:
You can run `yarn lint --fix` if there are errors. I realize that most problems it can't actually fix. It seems appropriate to keep the fix as a manual step.